docs/pages/includes/enterprise/samlauthentication.mdx

```diff
@@ -1,57 +1,58 @@
+### Enable default SAML authentication
 
-- Configure Teleport to use SAML authentication as the default instead of the local
+Configure Teleport to use SAML authentication as the default instead of the local
 user database.
 
-  <ScopedBlock scope={["oss", "enterprise"]}>
-  You can either edit the Teleport Auth Service configuration file or create a dynamic
-  resource.
-  </ScopedBlock>
+<ScopedBlock scope={["oss", "enterprise"]}>
+You can either edit the Teleport Auth Service configuration file or create a dynamic
+resource.
+</ScopedBlock>
 
-  <Tabs>
-  <TabItem scope={["cloud", "oss", "enterprise"]} label="Dynamic Resources (All Editions)">
+<Tabs>
+<TabItem scope={["cloud", "oss", "enterprise"]} label="Dynamic Resources (All Editions)">
```

Comment:
I'm not sure if the scope switcher will work correctly if we have multiple tab items that share the same scope. Maybe remove the `oss` and `enterprise` scopes here? We'll also want to add a `team` scope to this tab item